pkg/k8shandler/visualization.go

```diff
@@ -432,32 +436,75 @@ func (clusterRequest *ClusterLoggingRequest) createOrUpdateKibanaConsoleExternal
 		return err
 	}
 
-	consoleExternalLogLink := NewConsoleExternalLogLink(
-		"kibana",
+	consoleExternalLogLinkForAdmin := NewConsoleExternalLogLink(
+		"kibana-admin",
```

Comment:
have we coordinated with the console team regarding the object name change yet?

Reply:
@spadgett does the name matter?  its applied using a regex for the namespace correct?

Reply:
Right, console doesn't look at name. We list all console log links and apply them based on the namespace regex.

Reply:
`kibana-non-admin` may be confusing for users... what about `kibana` and `kibana-admin` ?

Reply:
@ewolinetz I changed it to `kibana` and `kibana-admin`